cirq/ops/pauli_string_test.py

```diff
@@ -616,3 +616,27 @@ def test_with_qubits():
     for q in new_qubits:
         assert new_pauli_string[q] == cirq.Pauli.by_index(q.x)
     assert new_pauli_string.coefficient == -1
+
+
+@pytest.mark.parametrize('paulis, coefficient, expected_unitary', (
```

Comment:
Do we call the "is operation consistent" test on a Pauli string instance? If not, add it.